jdi-light/src/main/java/com/epam/jdi/light/settings/PageSettings.java

```diff
@@ -3,9 +3,49 @@
 import com.epam.jdi.light.common.PageChecks;
 import com.epam.jdi.light.elements.composite.WebPage;
 import com.epam.jdi.tools.func.JAction1;
+import java.lang.reflect.InvocationTargetException;
 
 public class PageSettings {
+
+    static {
+        TestLoaded.test("com.epam.jdi.light.elements.composite.WebPage");
+    }
+
     public PageChecks checkPageOpen = PageChecks.NONE;
-    public JAction1<WebPage> beforeNewPage = WebPage::beforeNewPage;
-    public JAction1<WebPage> beforeEachStep = WebPage::beforeThisPage;
+
+    static {
+        TestLoaded.test("com.epam.jdi.light.elements.composite.WebPage");
+    }
+
+    static {
+        TestLoaded.test("com.epam.jdi.light.elements.composite.WebPage");
+    }
+
+    public JAction1<WebPage> beforeNewPage = (x) -> WebPage.beforeNewPage(x);
+    public JAction1<WebPage> beforeEachStep = (x) -> WebPage.beforeThisPage(x);
+
+    public static class TestLoaded {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'TestLoaded' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5408219)